joern-cli/frontends/php2cpg/src/test/scala/io/joern/php2cpg/querying/ControlStructureTests.scala

```diff
@@ -0,0 +1,436 @@
+package io.joern.php2cpg.querying
+
+import io.joern.php2cpg.astcreation.AstCreator.TypeConstants
+import io.joern.php2cpg.testfixtures.PhpCode2CpgFixture
+import io.shiftleft.codepropertygraph.generated.ControlStructureTypes
+import io.shiftleft.codepropertygraph.generated.nodes.{Block, ControlStructure, Identifier, Literal}
+import io.shiftleft.semanticcpg.language._
+
+class ControlStructureTests extends PhpCode2CpgFixture {
+
+  "if statements" should {
+    "work without a body, an elseif or an else" in {
+      val cpg = code("<?php\nif ($a) {}")
+
+      val ifAst = cpg.controlStructure.l match {
+        case List(ast) =>
+          ast.controlStructureType shouldBe ControlStructureTypes.IF
+          ast
+
+        case result => fail(s"Expected single control structure but found $result")
```

Comment:
You could also use Scalatests `Inside` trait and `inside` function to save you from writing the fail case manually.

Reply:
Ah, that's very useful! Thanks!!